index/btree.go

```diff
@@ -64,3 +64,23 @@ func (mt *MemoryBTree) Delete(key []byte) (*wal.ChunkPosition, bool) {
 func (mt *MemoryBTree) Size() int {
 	return mt.tree.Len()
 }
+
+// Ascend calls f for each item in the tree in ascending order. If f returns false, iteration stops.
```

Comment:
`f -> handleFunc/handleFn` maybe more clear?

The parameter of the function is not ChunkPosition, but the value in the file, we must read the value according to the position.

Reply:
Based on your suggestion, I have made further modifications to the code. Could you please review it to see if it meets your expectations? If not, please point out any issues, and I will make further adjustments. Due to my daytime commitments for company projects, I can only work on the open-source project during the evenings. I apologize for any delay in responding.

Reply:
it`s so kind of you to tell me this, I also maintain the project in my spare time, so just feel free to contribute.

Reply:
Thank you very much for the project you've written, which has enabled me to further study the Bitcask model.